rust/htp/src/decompressors.rs

```diff
@@ -897,6 +937,7 @@ impl Decompress for InnerDecompressor {
                 HtpContentEncoding::Deflate => HtpContentEncoding::Zlib,
                 HtpContentEncoding::Zlib => HtpContentEncoding::Gzip,
                 HtpContentEncoding::Lzma => HtpContentEncoding::Deflate,
+                HtpContentEncoding::Brotli => HtpContentEncoding::Deflate,
```

Comment:
can this have a comment about why we map brotli to deflate here?

Reply:
Sure, the idea is to retry the 3 classic deflate, zlib and gzip if we failed for other encodings like lzma, brolti (or zstd in the future)